tracer/src/Datadog.Trace.Tools.Runner/Checks/ProcessBasicCheck.cs

```diff
@@ -81,9 +84,52 @@ public static bool Run(ProcessInfo process)
                 foundIssue = true;
             }
 
+            if (!CheckRegistry())
```

Comment:
This will still get called by `dd-trace check process` command correct? Would this be better suited for a `dd-trace check environment` command? Or perhaps all subcommands (e.g. a future `iis`) would check the registry as part of their checks?

Reply:
> Or perhaps all subcommands (e.g. a future iis) would check the registry as part of their checks?

That's my plan. Every subcommand should perform the subset of checks relevant to the target they're testing. I don't think overlaps are an issue.